splashsurf_lib/src/density_map.rs

```diff
@@ -108,6 +108,12 @@ pub enum DensityMap<I: Index, R: Real> {
     DashMap(ReadDashMap<I, R, HashState>),
 }
 
+impl<I: Index, R: Real> Default for DensityMap<I, R> {
+    fn default() -> Self {
+        Self::Standard(Default::default())
+    }
+}
+
```

Comment:
I think I would just leave out the Default implementation and instead use an Option<...> field in the `SurfaceReconstruction` type. When I implement another reconstruction strategy, it might not return a density map, and a user-facing Default implementation does not look useful to me.